R/pkg/R/functions.R

```diff
@@ -3115,6 +3166,11 @@ setMethod("dense_rank",
 #'
 #' This is equivalent to the LAG function in SQL.
 #'
+#' @param x the column as a character string or a Column to compute on.
+#' @param offset the number of rows back from the current row from which to obtain a value.
+#'               If not specified, the default is 1.
+#' @param defaultValue default to use when the offset row does not exist.
+#' @param ... further arguments to be passed to or from other methods.
```

Comment:
This is actually the issue I raised at the end of #14558. In the same doc, it also includes generic definition, which also comes with `...` and this has the actual meaning "further arguments to be passed".